mitosheet/src/mito/components/toolbar/Toolbar.tsx

```diff
@@ -84,6 +85,7 @@ export const Toolbar = (
     }
     if (props.uiState.currOpenTaskpane.type === TaskpaneType.GRAPH) {
         tabs['Chart Design'] = <ChartDesignTabContents {...props} />
+        tabs['Format'] = <ChartFormatTabContents {...props} />
```

Comment:
When these are included, let's highlight the text in purple like you have in the mockups to make sure users see them. 

<img width="80" alt="Screenshot 2024-01-21 at 4 04 11 PM" src="https://github.com/mito-ds/mito/assets/18709905/8d1e91fa-d73b-4118-97c7-0f3c152c4cfa">